gems/pending/appliance_console/database_replication.rb

```diff
@@ -0,0 +1,116 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'English'
+
+module ApplianceConsole
+  class DatabaseReplication
+    include ApplianceConsole::Logging
+
+    REPMGR_CONFIG = '/etc/repmgr.conf'.freeze
+
+    attr_accessor :cluster_name, :node_number,
+                  :primary_database, :primary_database_username, :primary_database_password,
+                  :primary_host, :standby_host
+
+    def ask_for_cluster_name
+      self.cluster_name = just_ask("cluster name", cluster_name)
```

Comment:
I don't think we should be asking for a cluster name. If we're configuring a primary then the database has to be configured already so we should just use something static that incorporates the region number (`miq_region_0_cluster`?). Then on the standby side we can detect the primary database region number using the given credentials and primary database hostname.

Naming things is hard so let's do it for the user instead of making them come up with a name for something that doesn't really matter.